tests/integration/query/simple/with_filter/with_gt_datetime_test.go

```diff
@@ -29,12 +29,12 @@ func TestQuerySimpleWithDateTimeGTFilterBlockWithEqualValue(t *testing.T) {
 				`{
 					"Name": "John",
 					"Age": 21,
-					"CreatedAt": "2017-07-23T03:46:56.647Z"
+					"CreatedAt": "2017-07-23T03:46:56-05:00"
 				}`,
 				`{
 					"Name": "Bob",
 					"Age": 32,
-					"CreatedAt": "2010-07-23T03:46:56.647Z"
+					"CreatedAt": "2010-07-23T03:46:56-05:00"
```

Comment:
question: Why have these changed?  Do we no longer support UTC?

Reply:
We support UTC. `-05:00` is just to use the proper format that that John selected previously which is `time.RFC3339`.

Reply:
A `Z` suffix still works then?  Do we have tests that still test with the `Z` suffix?

Reply:
Just made one use it so it's covered 👍 Will show up once I commit changes.

Reply:
Cheers Fred